src/modules/services/submodule/assets/views/ViewServicesAssetsCreate.vue

```diff
@@ -400,6 +424,16 @@ export default {
         cosignerAddress.value = n.cosignerList.length>0?getMultiSigCosigner.value.cosignerList[0].address:''
       }
     })
+
+    const findAccWithAddress = address =>{
+      return walletState.currentLoggedInWallet.accounts.find(acc=>acc.address==address)
+    }
+
```

Comment:
walletState.currentLoggedInWallet can be null, the app will hang when user clicks log out, please return null if it's null

Reply:
noted, will revise it